src/integrationTest/java/com/hivemq/cli/commands/cli/TestBrokerCommandDefaultIT.java

```diff
@@ -18,38 +18,29 @@
 
 import com.hivemq.cli.MqttCLIMain;
 import com.hivemq.cli.utils.TestLoggerUtils;
-import com.hivemq.testcontainer.junit5.HiveMQTestContainerExtension;
+import io.github.sgtsilvio.gradle.oci.junit.jupiter.OciImages;
 import org.jetbrains.annotations.NotNull;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
-import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.hivemq.HiveMQContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @Disabled("Tests are only used to check output")
+@Testcontainers
 class TestBrokerCommandDefaultIT {
 
-    private static final @NotNull HiveMQTestContainerExtension hivemq =
-            new HiveMQTestContainerExtension(DockerImageName.parse("hivemq/hivemq4"));
-
-    @BeforeAll
-    static void beforeAll() {
-        hivemq.start();
-    }
+    @Container
+    private final @NotNull HiveMQContainer hivemq = new HiveMQContainer(OciImages.getImageName("hivemq/hivemq4"));
```

Comment:
Before, the hivemq container was used for all tests in this class (started beforeAll, stopped afterAll).
Now, every test gets a new hivemq container.
This is the case for multiple tests (I will not duplicate this comment to all occurrences).

Reply:
This was changed on purpose to remove static objects.